dotcom-rendering/src/components/AppsLightboxImage.importable.tsx

```diff
@@ -79,6 +79,7 @@ export const AppsLightboxImage = ({
 				background: none;
 				padding: 0;
 				width: 100%;
+				height: 100%;
```

Comment:
Fixes this bug

<img width="457" alt="image" src="https://github.com/guardian/dotcom-rendering/assets/19683595/3ac3b09f-8ff3-43d8-9c9e-6ac591f682cd">